service/controller/chart/v1/resource/release/create.go

```diff
@@ -90,6 +90,13 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 		case <-ch:
 			// Fall through.
 		case <-time.After(3 * time.Second):
+			// We set the checksum annotation so the update state calculation
+			// is accurate when we check in the next reconciliation loop.
+			err = r.patchAnnotations(ctx, cr, releaseState)
+			if err != nil {
+				return microerror.Mask(err)
+			}
```

Comment:
This got missed from https://github.com/giantswarm/chart-operator/pull/362.

We need to always set the checksum annotation so the update state calculation is accurate in the next loop. Currently we're triggering too many updates.